sdk/src/main/scala/io/horizen/certificatesubmitter/strategies/CircuitStrategy.scala

```diff
@@ -25,7 +26,7 @@ abstract class CircuitStrategy[
   PM <: SidechainBlockBase[TX, H],
   HIS <: AbstractHistory[TX, H, PM, _, _, _],
   MS <: AbstractState[TX, H, PM, MS],
-  T <: CertificateData](settings: SidechainSettings, params: NetworkParams) extends SparkzLogging{
+  T <: CertificateData](settings: SidechainSettings, sc2scConfig: Sc2ScConfigurator, params: NetworkParams) extends SparkzLogging with Sc2ScUtils[TX, H, PM, MS, HIS] {
```

Comment:
no need to extend `Sc2ScUtils` here.
Must be used only by CircuitWithKeyRotation